extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../../src/services';
+import {
+  AmpAdNetworkDoubleclickImpl,
+  resetRtcStateForTesting,
+} from '../amp-ad-network-doubleclick-impl';
+import {createElementWithAttributes} from '../../../../src/dom';
+import {installDocService} from '../../../../src/service/ampdoc-impl';
+import {Xhr} from '../../../../src/service/xhr-impl';
+
+// Uncomment out the next line when testing locally
+ import {AmpAd} from '../../../amp-ad/0.1/amp-ad';
+
+describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
+  let impl;
+  let element;
+  let sandbox;
+  let xhrMock;
+
+  beforeEach(() => {
+    sandbox = env.sandbox;
+    env.win.AMP_MODE.test = true;
+    resetRtcStateForTesting();
```

Comment:
I like having it there because it assures that my test can't break due to someone else not calling the reset in their test. It doesn't hurt any performance or anything, and helps with dealing with leakage from other tests.